src/base/config_file_stream.cc

```diff
@@ -218,4 +222,20 @@ std::string ConfigFileStream::GetFileName(const absl::string_view filename) {
 void ConfigFileStream::ClearOnMemoryFiles() {
   Singleton<OnMemoryFileMap>::get()->clear();
 }
+
+#ifdef _WIN32
+void ConfigFileStream::FixupFilePermission(absl::string_view filename) {
+    const auto path = ConfigFileStream::GetFileName(filename);
```

Comment:
Let's use `std::string` instead of `auto`.

Reply:
Sorry, forgot to update it. Done in 353599a0a1015887c400ad3c0fb0198d978729c6.